src/client/datascience/jupyter/kernels/types.ts

```diff
@@ -169,8 +168,7 @@ export interface IKernel extends IAsyncDisposable {
     readonly info?: KernelMessage.IInfoReplyMsg['content'];
     readonly kernelSocket: Observable<KernelSocketInformation | undefined>;
     start(options?: { disableUI?: boolean }): Promise<void>;
-    interruptCell(cell: NotebookCell): Promise<InterruptResult>;
-    interruptAllCells(document: NotebookDocument): Promise<InterruptResult>;
+    interrupt(document: NotebookDocument): Promise<InterruptResult>;
```

Comment:
No more cancelCell, cancelAllCells, interruptCell, interruptAllCells - they all mean interrupt in Jupyter world.